Library/Homebrew/download_strategy.rb

```diff
@@ -312,6 +329,10 @@ def _fetch
     curl @url, "-C", downloaded_size, "-o", temporary_path
   end
 
+  def does_redirect
```

Comment:
Might want to make this `does_redirect?` and return a true or false value based on the `@url`